en/docs/install-and-setup/upgrading-wso2-api-manager/upgrading-from-200-to-310.md

```diff
@@ -470,7 +470,7 @@ Follow the instructions below to move all the existing API Manager configuration
 3.  To start the migration process, run the respective migration script based on your environment.
 
     ??? note "Linux/Mac OS"
-        Run the [apim200_to_apim300_gateway_artifact_migrator.sh](../../assets/attachments/install-and-setup/apim200_to_apim300_gateway_artifact_migrator.sh) script, as shown below, to migrate from WSO2 API Manager 2.0.0 to 3.0.0. 
+        Run the [apim200_to_apim300_gateway_artifact_migrator.sh](../../assets/attachments/install-and-setup/apim200_to_apim300_gateway_artifact_migrator.sh) script, as shown below, to migrate from WSO2 API Manager 2.0.0 to 3.1.0. 
```

Comment:
Shall we rename the script file name?

Reply:
+1

Reply:
Adding fd6185537f170e635be231dd5f58721f09d8ae6d